src/main/java/com/dreammaster/gthandler/recipes/FluidSolidifierRecipes.java

```diff
@@ -25,543 +28,468 @@ public class FluidSolidifierRecipes implements Runnable {
-                GT_ModHandler.getModItem(GregTech.ID, "gt.metaitem.02", 1L, 32421),
-                100,
-                16);
-        GT_Values.RA.addFluidSolidifierRecipe(
-                ItemList.Shape_Mold_Ball.get(0L),
-                new FluidStack(FluidRegistry.getFluid("dye.chemical.dyegray"), 72),
-                GT_ModHandler.getModItem(GregTech.ID, "gt.metaitem.02", 1L, 32422),
-                100,
-                16);
-        GT_Values.RA.addFluidSolidifierRecipe(
-                ItemList.Shape_Mold_Ball.get(0L),
-                new FluidStack(FluidRegistry.getFluid("dye.chemical.dyepink"), 72),
-                GT_ModHandler.getModItem(GregTech.ID, "gt.metaitem.02", 1L, 32423),
-                100,
-                16);
-        GT_Values.RA.addFluidSolidifierRecipe(
-                ItemList.Shape_Mold_Ball.get(0L),
-                new FluidStack(FluidRegistry.getFluid("dye.chemical.dyelime"), 72),
-                GT_ModHandler.getModItem(GregTech.ID, "gt.metaitem.02", 1L, 32424),
-                100,
-                16);
-        GT_Values.RA.addFluidSolidifierRecipe(
-                ItemList.Shape_Mold_Ball.get(0L),
-                new FluidStack(FluidRegistry.getFluid("dye.chemical.dyeyellow"), 72),
-                GT_ModHandler.getModItem(GregTech.ID, "gt.metaitem.02", 1L, 32425),
-                100,
-                16);
-        GT_Values.RA.addFluidSolidifierRecipe(
-                ItemList.Shape_Mold_Ball.get(0L),
-                new FluidStack(FluidRegistry.getFluid("dye.chemical.dyelightblue"), 72),
-                GT_ModHandler.getModItem(GregTech.ID, "gt.metaitem.02", 1L, 32426),
-                100,
-                16);
-        GT_Values.RA.addFluidSolidifierRecipe(
-                ItemList.Shape_Mold_Ball.get(0L),
-                new FluidStack(FluidRegistry.getFluid("dye.chemical.dyemagenta"), 72),
-                GT_ModHandler.getModItem(GregTech.ID, "gt.metaitem.02", 1L, 32427),
-                100,
-                16);
-        GT_Values.RA.addFluidSolidifierRecipe(
-                ItemList.Shape_Mold_Ball.get(0L),
-                new FluidStack(FluidRegistry.getFluid("dye.chemical.dyeorange"), 72),
-                GT_ModHandler.getModItem(GregTech.ID, "gt.metaitem.02", 1L, 32428),
-                100,
-                16);
-        GT_Values.RA.addFluidSolidifierRecipe(
-                ItemList.Shape_Mold_Ball.get(0L),
-                new FluidStack(FluidRegistry.getFluid("dye.chemical.dyewhite"), 72),
-                GT_ModHandler.getModItem(GregTech.ID, "gt.metaitem.02", 1L, 32429),
-                100,
-                16);
+
+        GT_Values.RA.stdBuilder().itemInputs(ItemList.Shape_Mold_Ball.get(0L))
+                .itemOutputs(GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Sodium, 1L))
+                .fluidInputs(Materials.Sodium.getFluid(1000)).noFluidOutputs().duration(20 * TICKS).eut(4)
+                .addTo(sFluidExtractionRecipes);
```

Comment:
this one is more major: all of these need to be sFluidSolidficationRecipes.

Reply:
whoopsie, will fix it in a second